.github/workflows/deploy.yml

```diff
@@ -50,14 +52,13 @@ jobs:
         run: |
           echo "PATH=$(pwd)/node_modules/.bin/:$PATH" >> $GITHUB_ENV
       - name: run unit tests
-        run: ./test-unit.sh {{ env.STAGE_PREFIX }}{{ env.branch_name }}
+        run: ./test-unit.sh
       - name: publish test coverage to code climate
         if: env.CODE_CLIMATE_ID != ''
         uses: paambaati/codeclimate-action@v2.7.5
         env:
           CC_TEST_REPORTER_ID: ${{ secrets.CODE_CLIMATE_ID }}
         with:
-          debug: true
```

Comment:
Turn off the debug log level for this action, nothing interesting is happening... just to declutter logs.